zipkin-lens/src/components/UiConfig/UiConfig.jsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import useFetch from 'fetch-suspense';
+import React from 'react';
+
+import { UI_CONFIG } from '../../constants/api';
+
+const ConfigContext = React.createContext();
+
+export const UiConfig = ({ children }) => {
+  const response = useFetch(UI_CONFIG, { lifespan: 1 });
```

Comment:
lifespan 1?

Reply:
The library I tried using for fetching with suspense defaults to caching responses which we didn't need (hence lifetime of 1ms). But the library itself seemed sketchy (doesn't really follow the best practices for suspense), and many lines more than the simple wrapper I wrote instead so just got rid of it.